src/script/bitcoinconsensus.cpp

```diff
@@ -13,13 +13,13 @@
 namespace {
 
 /** A class that deserializes a single CTransaction one time. */
-class TxInputStream
+class InputStream
```

Comment:
I would advice introducing the code style changes in their own commit. Also, I think it's clearer to keep the `Tx` prefix, you have other class of inputs in the codebase beyond transaction like hash elements.